docs/Migrating.md

```diff
@@ -1,5 +1,14 @@
 # Migrating
 
+## Migrating from master to develop
+
+### Important changes
+* `CustomResetParameters` are now removed.
+* `reset()` on the Low-Level Python API no longer takes a `train_mode` argument. To modify the performance/speed of the engine, you must use an `EngineConfigurationChannel`
+* `reset()` on the Low-Level Python API no longer takes a `config` argument. `UnityEnvironment` no longer has a `reset_parameters` field. To modify float properties in the environment, you must use a `FloatPropertiesChannel`
+* The Academy no longer has a `Training Configuration` nor `Inference Configuration` field in the inspector. To modify the configuration form the Low-Level Python API, use an `EngineConfigurationChannel`. To modify it during training, use the new command line arguments `--width`, `--heigth`, `--quality-level`, `--time-scale` and `--target-frame-rate` in `mlagents-learn`.
```

Comment:
Maybe something like "These default to the previous default values of the Training Configuration fields. If you weren't using `--slow`, you don't need to change anything. If you were running training with `--slow`, you should now pass `--quality-level=... etc`" (it might be better in a "Steps to Migrate" section" with slightly different wording)

Reply:
I added : 

### Steps to Migrate
 * If you had a custom `Training Configuration` in the Academy inspector, you will need to pass your custom configuration at every training run using the new command line arguments `--width`, `--heigth`, `--quality-level`, `--time-scale` and `--target-frame-rate`.
 * If you were using `--slow` in `mlagents-learn`, you will need to pass your old `Inference Configuration` of the Academy inspector with the new command line arguments `--width`, `--heigth`, `--quality-level`, `--time-scale` and `--target-frame-rate` instead.